src/device-registry/config/global/mappings-v1.js

```
@@ -0,0 +1,599 @@
+    field8: "ExtraData",
+  },
+  THINGSPEAK_GAS_FIELD_DESCRIPTIONS: {
+    field1: "PM2.5",
+    field2: "TVOC",
+    field3: "HCHO",
+    field4: "CO2",
+    field5: "Intake Temperature",
+    field6: "Intake Humidity",
+    field7: "Battery Voltage",
+    field8: "ExtraData",
+  },
```

Biome (1.9.4)

[error] This property value named THINGSPEAK_GAS_FIELD_DESCRIPTIONS is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named THINGSPEAK_GAS_FIELD_DESCRIPTIONS


(lint/suspicious/noDuplicateObjectKeys)